website/content/docs/k8s/helm.mdx

```diff
@@ -689,20 +679,6 @@ Use these links to navigate to a particular top-level stanza.
     Note: if running on OpenShift, this setting is ignored because the user and group are set automatically
     by the OpenShift platform.
 
-<<<<<<< HEAD
-=======
-  - `containerSecurityContext` ((#v-client-containersecuritycontext)) (`map`) - The container securityContext for each container in the client pods. In
-    addition to the Pod's SecurityContext this can
-    set the capabilities of processes running in the container and ensure the
-    root file systems in the container is read-only.
-
-    - `client` ((#v-client-containersecuritycontext-client)) (`map`) - The consul client agent container
-
-    - `aclInit` ((#v-client-containersecuritycontext-aclinit)) (`map`) - The acl-init initContainer
-
-    - `tlsInit` ((#v-client-containersecuritycontext-tlsinit)) (`map`) - The tls-init initContainer
```

Comment:
These lines too

Reply:
Thanks for the sanity check @lkysow . Adding the now.